packages/psss-server/src/app/attachModels.ts

```diff
@@ -3,17 +3,19 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 import { Response, NextFunction } from 'express';
+import { ModelRegistry } from '@tupaia/database';
 import { PsssSessionModel } from '../models';
 import { PsssRequest } from '../types';
 
 /**
  * Attach psss session model to the request
  */
-export const attachSessionModel = (sessionModel: PsssSessionModel) => (
+export const attachModels = (models: ModelRegistry, sessionModel: PsssSessionModel) => (
   req: PsssRequest,
   res: Response,
   next: NextFunction,
 ) => {
+  req.models = models;
```

Comment:
Need this to get the entity records from the database in the endpoints. I also think we will need to use it more in the future

Reply:
The idea was that psss-server shouldn't have access to the database at all (session table was stored in the same database purely for convenience)